src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.Impl.cs

```diff
@@ -102,6 +102,17 @@ public int ZIndex
 			set { SetValue(ZIndexProperty, value); }
 		}
 
+		string ITooltip.TooltipText => TooltipText;
+
+		public static readonly BindableProperty TooltipTextProperty =
+ 			BindableProperty.Create(nameof(TooltipText), typeof(string), typeof(VisualElement), defaultValue: null);
+
+		public string TooltipText
```

Comment:
This is limiting API: We should take a `object`, do a ToString() for now.

Reply:
Done